### PR TITLE
Created contributing.md file

### LINKED ISSUE #32: [DOCS] Add contributing guidelines on how folks can add a youtube channel to the list.
We require someone to create a CONTRIBUTING.md file which states the guidelines on how folks can add a youtube channel to the list.

@Agrim-Sharma174 @vikasganiga05 - Please edit this description with further instructions (if any) to help the user make the PR.

### PR DESCRIPTION
**Pull request by AlphaDojo**
fixes #32 

Added a contributing file to show users how to add a new entry into the existing table. Edited the readme to let users know about the contributing file.

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Hi. Everything looks good but please update the things from Youtube to YouTube, I require the T caps as well. Thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Contribution
+
+## Link qualification
+Links have to contain Youtube content pertaining to coding. It has to be helpful, contain a short description as to why it's helpful.
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:4]:
```suggestion
Links have to contain YouTube content pertaining to coding. It has to be helpful, contain a short description as to why it's helpful.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Here are some rules to follow when adding a link to make sure it qualifies:
+1. Before proceeding with every step below, take your time to read the current list to see if the Youtuber you want to add hasn't already been added. (This saves the maintainer's time when reviewing your pull request.
+2. The link has to be a **Youtube** link.
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:7]:
```suggestion
2. The link has to be a **YouTube** link.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+3. Based on the point above, no links to the Youtuber's other accounts such as Twitter, Github, etc.
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:8]:
```suggestion
3. Based on the point above, no links to the YouTuber's other accounts such as Twitter, Github, etc.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+4. The content within the Youtube channel must be about code or coding.
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:9]:
```suggestion
4. The content within the YouTube channel must be about code or coding.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+5. Ensure that the content is objective and not opiniated. This means no resources that talks about someone's thoughts of a topic, even if you personally find it helpful.
+6. If you're a Youtuber making beginner-styled tutorial content on, you're allowed to post a link to your channel given it follows the other rules detailed above.
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:11]:
```suggestion
6. If you're a YouTuber making beginner-styled tutorial content on, you're allowed to post a link to your channel given it follows the other rules detailed above.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+## Adding a link
+Adding a Youtuber is straightforward!
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:13]:
```suggestion
Adding a YouTuber is straightforward!
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+1. Get the Youtuber channel link you want to add.
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:14]:
```suggestion
1. Get the YouTuber's channel link you want to add.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+2. Go to the [README](https://github.com/collab-community/coding-youtubers/blob/main/README.md)
+3. Click on the edit icon on the top right
+![](Assets/step3.png)
+4. Scroll to the bottom of the table and input your Youtube channel link with the format in mind.
+Format:
+```
+| [Youtube Channel Name](Youtube Channel Link) | Description | Language|
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:21]:
```suggestion
| [YouTube Channel Name](YouTube Channel Link) | Description | Language|
```

File: README.md [hunk 9]
```diff
@@ -2,9 +2,7 @@
 
 This repo is created, for the community to find Coding Youtubers, which will really help beginners start their coding journey.
 
-You can add a YouTuber that helped you begin coding in the below list by creating a PR.
-
-If you're a YouTuber reading this and if you think you make beginner-friendly tutorial videos, please add your name alongwith your YouTube channel link.
+You can add a YouTuber that helped you begin coding in the below list by following our contribution guide [here](CONTRIBUTING.md).
```
Comment (KendallDoesCoding) [inline README.md:5]:
```suggestion
You can add a YouTuber that helped you begin coding in the below list by following our CONTRIBUTING GUIDELINES [here](CONTRIBUTING.md).
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Comment by AlphaDojo** [comment]
Hi, I've just committed your suggestions. Please have a look.

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution!

**Pull request merged**